lib/atomic_web/components/announcement.ex

```diff
@@ -0,0 +1,47 @@
+defmodule AtomicWeb.Components.Announcement do
+  @moduledoc false
+  use AtomicWeb, :component
+
+  alias Atomic.Uploaders.Logo
+
+  def announcement(assigns) do
+    ~H"""
+    <%= live_redirect to: @url, class: "group" do %>
+      <li id={"announcement-#{@news.id}"} class="relative py-5 pr-6 pl-4 border-b border-gray-200 sm:py-6 sm:pl-6 lg:pl-8 xl:pl-6 hover:bg-gray-50">
+        <div>
+          <h3 class="text-sm font-semibold text-gray-800 hover:underline focus:outline-none">
+            <%= @news.title %>
+          </h3>
+          <article class="mt-1 text-sm text-gray-600 line-clamp-3">
+            <dd class="text-sm text-gray-500"><%= @news.description %></dd>
```

Comment:
```suggestion
            <dd class="text-sm text-gray-500"><%= maybe_slice_string(news.description, 250) %></dd>
```
Choose a number that fits well. 250 is random 😅